build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,334 @@
+    # from that range will be assigned whenever a Pod's container defines a specific port to be exposed
+    # (each container can define a list of ports as pod.spec.containers[].ports), and all Node traffic
+    # directed to that port will be forwarded to the Pod.
+    #  portRange: 61000-62000
+  antrea-cni.conflist: |
+    {
+        "cniVersion":"0.3.0",
+        "name": "antrea",
+        "plugins": [
+            {
+                "type": "antrea",
+                "ipam": {
+                    "type": "host-local"
+                },
+                "capabilities": {"dns": true}
+            }
+        ]
+    }
+kind: ConfigMap
+metadata:
+  labels:
+    app: antrea
+  name: antrea-windows-config-8kfkb8t957
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: antrea
+    component: antrea-agent
+  name: antrea-agent-windows
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-agent
+  template:
+    metadata:
+      annotations:
+        "microsoft.com/hostprocess-inherit-user": "true"
+      labels:
+        app: antrea
+        component: antrea-agent
+    spec:
+      securityContext:
+        windowsOptions:
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+          hostProcess: true
+      containers:
+        - name: antrea-agent
+          image: antrea/antrea-windows:latest
+          imagePullPolicy: IfNotPresent
+          args:
+            - -file
+            - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Run-AntreaAgent-OVS-Containerd.ps1
+          command:
+          - powershell
+          env:
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          volumeMounts:
+          - mountPath: /etc/antrea
+            name: antrea-windows-config
+          - mountPath: /var/lib/antrea-windows
+            name: antrea-agent-windows
+          - mountPath: /var/log/antrea/
+            name: var-log-antrea
+        - name: antrea-ovs
+          image: antrea/windows-ovs:1809-3.0.5
+          imagePullPolicy: IfNotPresent
+          args:
+            - -file
+            - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Run-AntreaOVS-Containerd.ps1
+          command:
+          - powershell
+          volumeMounts:
+          - mountPath: /var/lib/antrea-windows
+            name: antrea-agent-windows
+          - mountPath: /var/log/openvswitch
+            name: var-log-antrea
+            subPath: openvswitch
+      hostNetwork: true
+      initContainers:
+      - args:
+        - -File
```

Comment:
I see both `-file` and `-File` in the yaml, please check and make them consistent if both works.